scripts/prepare_network.py

```diff
@@ -70,21 +70,14 @@
 logger = logging.getLogger(__name__)
 
 
-def add_co2limit(n, Nyears=1., factor=None):
-
-    if factor is not None:
-        annual_emissions = factor*snakemake.config['electricity']['co2base']
-    else:
-        annual_emissions = snakemake.config['electricity']['co2limit']
+def add_co2limit(n, co2limit=1.487e+9, Nyears=1.):
```

Comment:
I would remove the default for `co2limit`. As it assumes some spatial scope of the model.

Reply:
Agree, but I think that shouldn't be done in this PR because we're improving style not content here

Reply:
But you are adding this hard coded number here. This would just as well function without a default because it is never called as `add_co2limit(n)`:

```py
def add_co2limit(n, co2limit, Nyears=1.)
```